.github/workflows/workflow-validate-website-content.yml

```
@@ -5,7 +5,7 @@ on:
       site_git_ref:
         description: 'git ref, a SHA commit or branch or tag, for the published HTML content'
         required: true
-        default: 'nist-pages'
+        default: 'github-pages'
```

actionlint (1.7.4)

input "site_git_ref" of workflow_call event has the default value "github-pages", but it is also required. if an input is marked as required, its default value will never be used

(events)